common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -56,6 +55,12 @@
         <item name="textAppearanceH3">@style/Typography.DuckDuckGo.H3</item>
         <item name="textAppearanceH4">@style/Typography.DuckDuckGo.H4</item>
         <item name="textAppearanceH5">@style/Typography.DuckDuckGo.H5</item>
+        <item name="textAppearanceButton">@style/Typography.DuckDuckGo.Button</item>
+
+        <!-- Shape -->
+        <item name="shapeAppearanceSmallComponent">@style/ShapeAppearance.DuckDuckGo.SmallComponent</item>
+        <item name="shapeAppearanceMediumComponent">@style/ShapeAppearance.DuckDuckGo.MediumComponent</item>
+        <item name="shapeAppearanceLargeComponent">@style/ShapeAppearance.DuckDuckGo.LargeComponent</item>
```

Comment:
I've checked and we are only using `shapeAppearanceSmallComponent`. You are setting button size in DaxButton class directly so not sure if this shapeAppearance should be applied there or just remove the unused styles and use general `shapeappearanceComponent`

Reply:
The shape is needed for the rounded background. The size only affects the height. I'd prefer keeping the three shapes since it's what Material expects.